main.cpp

```diff
@@ -63,9 +63,30 @@ video: Chapter 2 - Part 3
 void variableDeclarations()
 {
     //example:
-    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
+    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
     
-
+    int life = 0;
+    int myPower = 9;
+    int endOfRainbowLocation = 3;
+    
+    float hate = 66.f;
+    float airQuality = 1.f;
+    float hairThickness = 12.f;
+    
+    double blue = 0.21;
+    double speed = 51.6;
+    double drumTuning = 84.2;
+
+    char Z = 'Z';
+    char D = 'D';
+    char H = 'H';
+    
+    bool hasTail = true;
+    bool isMale = false;
+    bool isExpensive = true;
+{
+    ignoreUnused(life, myPower, endOfRainbowLocation, hate, airQuality, hairThickness, blue, speed, drumTuning, Z, D, H, hasTail, isMale, isExpensive);
+}

```

Comment:
Curly braces are not needed here.
you aren't calling `ignoreUnused` once in this function.  you're calling it twice.
The instructions tell you to call it once. 
- [ ] revise